src/js/server/components/ConsentLite.tsx

```diff
@@ -87,6 +103,7 @@ const ConsentLite = ({
 	isSubsection,
 	formOfWords,
 	showSubmitButton,
+	hideFooter,
```

Comment:
Question: I'm not sure what is best, I think we usually favour the showSomething flag rather than hide, it seems more intuitive...
But there is nothing wrong with this.
Also more consistent with `showSubmitButton`?

```suggestion
	showFooter = true,
```

Reply:
agree, changed! 2b28e6facceeea0011e37d52c7d2efad0b6991eb